src/lib/stores/player.ts

```diff
@@ -69,6 +86,21 @@ export function createPlayerStore() {
 		set(createInit());
 	};
 
+	const handlePaused = (paused: boolean) => {
+		if (paused) {
```

Comment:
why not keep the timout interval also on pause and just don't increment the state on every interval tick